src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2895,4 +2895,109 @@ public void startFakeRepairSchedule() {
+    synchronized (BasicSteps.class) {
+      testContext.TEST_CLUSTER = (String) clusterName;
+      Set<String> tables = Sets.newHashSet();
+      testContext.addClusterInfo(clusterName, keyspace, tables);
+      HashMap<String, String> params = Maps.newHashMap();
+      params.put("clusterName", clusterName);
+      params.put("keyspace", keyspace);
+      params.put("owner", "test_user");
+
+      for (int iter = 1; iter <= repairsAdded; iter++) {
+        Response response = RUNNERS.get(0).callReaper("POST", "/repair_run", Optional.of(params));
+        String responseData = response.readEntity(String.class);
+        Assertions
+            .assertThat(response.getStatus())
+            .isEqualTo(Response.Status.CREATED.getStatusCode())
+            .withFailMessage(responseData);
+        UUID id = UUID.randomUUID();
+        try {
+          RepairRunStatus repairRun = new ObjectMapper().readValue(responseData, RepairRunStatus.class);
+          id = repairRun.getId();
+          testContext.addCurrentRepairId(id);
+        } catch (Throwable e) {
+          LOG.error("response deserialisation failed", e);
+          LOG.error("Response data was: {}", responseData);
+          Assertions.fail("response deserialisation failed");
+        }
+        response = RUNNERS.get(0).callReaper(
+            "PUT",
+            String.format("repair_run/%s/state/%s", id.toString(), "RUNNING"),
+            Optional.empty());
+        Assertions
+            .assertThat(response.getStatus())
+            .isEqualTo(Response.Status.OK.getStatusCode())
+            .withFailMessage(responseData);
+        if (iter > (repairsAdded - repairsAborted)) {
+          response = RUNNERS.get(0).callReaper(
+              "PUT",
+              String.format("repair_run/%s/state/%s", id.toString(), "ABORTED"),
+              Optional.empty());
+          Assertions
+              .assertThat(response.getStatus())
+              .isEqualTo(Response.Status.OK.getStatusCode())
+              .withFailMessage(responseData);
+        } else {
+          response = RUNNERS.get(0).callReaper(
+              "PUT",
+              String.format("repair_run/%s/state/%s", id.toString(), "PAUSED"),
+              Optional.empty());
+          Assertions
+              .assertThat(response.getStatus())
+              .isEqualTo(Response.Status.OK.getStatusCode())
+              .withFailMessage(responseData);
+        }
+      };
+    }
+  }
+
+  @Then("^when I list the last (\\d+) repairs, I can see (\\d+) repairs at (\\d+) state$")
+  public void listRepairs(Integer limit, Integer expectedRepairsCount, String expectedState) {
+    synchronized (BasicSteps.class) {
+      RUNNERS.parallelStream().forEach(runner -> {
+        HashMap<String, String> params = Maps.newHashMap();
+        params.put("limit", limit.toString());
+        Response resp = runner.callReaper(
+            "GET",
+            "/repair_run/cluster/" + TestContext.TEST_CLUSTER,
```

Comment:
You may want to test `"/repair_run"` to cover both endpoints that allow listing repairs.

Reply:
Also done, and appears to be passing.